
class SlobypyException(Exception):
    pass


class AlreadyExistsException(SlobypyException):
    pass